package.json

```diff
@@ -37,6 +37,7 @@
     "compression": "^1.7.4",
     "express": "^4.17.1",
     "lit": "^2.0.2",
+    "minecraft-inventory-gui": "github:AwesomestCode/prismarine-inventory-gui",
```

Comment:
can you instead depend on the released version of this ?

Reply:
No; the released version doesn’t have all the features I need.

Reply:
ok, then PR the features you need upstream
then you will be able to use the released version here

Reply:
sorry for asking but what does PR mean?

Reply:
pull request

Reply:
this still need to be updated to the npm version of minecraft-inventory-gui ; so you'd need to PR any change you made to the repo there